src/views/BrowseView/BrowseView.js

```diff
@@ -102,6 +109,7 @@ const BrowseView = ({
       hasPrevPage={hasPrevPage}
       hidePageIndices
       resultsContainerRef={resultsContainerRef}
+      tenantId={tenantId}

```

Comment:
Do we need a PR in stripes-authority-components too? I can't find how this prop is used

Reply:
Yes, we need it, here it is https://github.com/folio-org/stripes-authority-components/pull/100